File: sdks/php/src/Types/SubscriptionResponseStatus.php
<?php

namespace NovaBilling\Types;

enum SubscriptionResponseStatus: string
{
    case Active = "ACTIVE";
    case PastDue = "PAST_DUE";
    case Canceled = "CANCELED";
    case Trialing = "TRIALING";
    case Paused = "PAUSED";
}

File: sdks/ruby/lib/novabilling/types/subscription_response_status.rb
# frozen_string_literal: true

module Novabilling
  module Types
    module SubscriptionResponseStatus
      extend Novabilling::Internal::Types::Enum

      ACTIVE = "ACTIVE"
      PAST_DUE = "PAST_DUE"
      CANCELED = "CANCELED"
      TRIALING = "TRIALING"
      PAUSED = "PAUSED"
    end
  end
end
